FrontEnd/src/components/ProfilePage/FormComponents/FormFields/HalfFormField.js

```diff
@@ -1,7 +1,14 @@
-import { PropTypes } from 'prop-types';
+import {PropTypes} from 'prop-types';
 import css from './HalfFormField.module.css';
+import {Tooltip} from 'antd';
+
+const LengthEmail = 32;
 
 const HalfFormField = (props) => {
+    const shouldShowTooltip = props.name === 'email' && props.value.length > LengthEmail;
+
+    const truncatedEmail = shouldShowTooltip ? `${props.value.slice(0, LengthEmail)}...` : props.value;
```

Comment:
If LengthEmail is set to 32 and the email length is longer, there is only one dot after truncated email instead of an ellipsis.
![image](https://github.com/ita-social-projects/Forum/assets/114805677/c7cc953a-e4d0-4622-82e5-239c8b3e9d88)

Reply:
I think you should rename variable since you use it for all the fields, like fieldValue.